recipes/cmiles/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "cmiles" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openforcefield/cmiles/archive/v0.1.5.tar.gz
+  sha256: 866e1d56ae4f86160fcbeeb782f688a6d4b1f14c88ed23341a74a1b3f2f04bbc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - rdkit
+
+test:
+  imports:
+    - cmiles
```

can we add ``pip check`` here please?
```suggestion
    - cmiles
  requires:
    - pip
  commands:
    - pip check
```